ansible_rulebook/rule_set_runner.py

```diff
@@ -157,7 +157,7 @@ async def _cleanup(self):
             )
 
         for task in self.active_actions:
-            logger.debug("Cancelling active task %s", task.get_name())
+            logger.info("Cancelling active task %s", task.get_name())
```

Comment:
@benthomasson and @mkanoor, [related comment](https://github.com/ansible/ansible-rulebook/pull/679/files#r1585503286): should we also reclassify all the log messages to `info` for the cleanup tasks?

Reply:
At one point we used to log all these messages at info level it was too much information so we ended up removing them. Should a summary message be more appropriate saying how many actions got canceled

Reply:
I left the loop message at `debug` and added a new `info` message with only the count of tasks to be cancelled.

Reply:
Start-up and clean-up messages are a finite amount of messages and are probably useful for end users.     I like that we log that we are pulling images in job history for instance.   AWX doesn't do that and users wonder why it is taking so long to start a job.